Userland/Libraries/LibTLS/Certificate.h

```diff
@@ -60,6 +60,8 @@ class Certificate {
     bool is_allowed_to_sign_certificate { false };
     bool is_certificate_authority { false };
     Optional<size_t> path_length_constraint {};
+    bool is_self_issued { false };
+    bool is_self_signed();
```

Comment:
Please add a newline after the last property and before this function declaration.

Reply:
Should I than just put it where `is_valid()` is?

Reply:
Sure, or just add a newline between the property and function?

Reply:
Done